application/language/spanish/qso_lang.php

```diff
@@ -7,11 +7,11 @@
 $lang['qso_title_suggestions'] = 'Sugerencias';
 $lang['qso_title_previous_contacts'] = 'Contactos previos';
 $lang['qso_title_times_worked_before'] = "times worked before";
-$lang['qso_title_image'] = 'Profile Picture';
-$lang['qso_previous_max_shown'] = "Max. 5 previous contacts are shown";
+$lang['qso_title_image'] = 'Imagen de Perfil';
+$lang['qso_previous_max_shown'] = "Se muestran máx. de 5 contactos previos";
```

Comment:
"muestra" instead of "muestran". It needs to match in number with the noun, which is "(un) máximo" here.